README.md

```diff
@@ -491,6 +491,7 @@ Name | Website | Region
 [PNC Financial Services ](/company-profiles/pnc-financial-services.md) | https://www.pnc.com/ | USA
 [Polygon](/company-profiles/polygon.md) | https://polygon.technology/careers/#all-roles | Worldwide
 [PowerSchool](/company-profiles/powerschool.md) | https://www.powerschool.com/ | North America / India
+[Pragma](/company-profiles/pragma.md) | https://www.pragma.co/ | Latin America | Colombia
```

Comment:
> [Pragma](/company-profiles/pragma.md) | https://www.pragma.co/ | Latin America | Colombia

Since this is a table, the | after `Latin America` may impact the build, I recommend you change it to Latin America, Colombia or even just Latin America as in the blurb, you say you are expanding offices